wavelength-plugins/tsumeta.js

```diff
@@ -115,6 +115,40 @@ exports.commands = {
 			write();
 		},
 
+		modify: "editproposal",
+		edit: "editproposal",
+		modifyproposal: "editproposal",
+		editproposal: function (target, room, user) {
+			if (!isTsuMetaCouncil(user.userid)) return this.errorReply('You are not in the TsuMeta council, or have been suspended.');
+			if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+			let parts = target.split(',');
+			for (let u in parts) parts[u] = parts[u].trim();
```

Comment:
Consolidate this into `let [proposal, ...newDesc] = target.split(',').map(p => p.trim());`. I never understand why panpawn doesn't use this...anyways, it allows you to remove `let proposal = parts[0]; let newDesc = parts[1];`, and it lets proposal descriptions include commas.